modules/ROOT/pages/monitoring/metrics/reference.adoc

```diff
@@ -494,6 +494,9 @@ label:deprecated[Deprecated in 5.15]
 |<prefix>.cluster.store_copy.pull_updates|The total number of pull requests made by this instance. (counter)
 |<prefix>.cluster.store_copy.pull_update_highest_tx_id_requested|The highest transaction id requested in a pull update by this instance. (counter)
 |<prefix>.cluster.store_copy.pull_update_highest_tx_id_received|The highest transaction id that has been pulled in the last pull updates by this instance. (counter)
+|<prefix>.cluster.store_copy.store_file_download_attempt|Store file download attempts. (counter)
+|<prefix>.cluster.store_copy.store_file_download_fail|Store file download failed downloads. (counter)
+|<prefix>.cluster.store_copy.store_file_download_success|Store file download successful downloads. (counter)
```

Comment:
```suggestion
|<prefix>.cluster.store_copy.store_file_download_success|The number of successful store file downloads. (counter)
```

Reply:
yeah, I agree. I was going to suggest something similar. @RagnarW, what do you think?